FunctionalEquivalence/FunctionalEquivalence.wdl

```diff
@@ -19,301 +36,257 @@ workflow FunctionalEquivalence {
 
         Array[String] truth_vcf
         Array[String] truth_vcf_index
+        Array[String] truth_vcf_sample_names
 
-        Array[File]? stratIntervals
-        Array[String]? stratLabels
+        Array[File]? stratifier_intervals
+        Array[String]? stratifier_labels
 
-        File reference
-        String referenceVersion = "hg38"
-        File refIndex
-        File refDict
-        File hapMap
+        File ref_fasta
+        File ref_index
+        File haplotype_map
 
         String tool1_label
         String tool2_label
         String? additional_label
 
+        # vcfeval Arguments
+        Boolean passing_only = true
+        Boolean require_matching_genotypes = true
+        Boolean enable_ref_overlap = false
+
         Boolean signed_difference = false
 
-        String gatkTag="4.0.11.0"
-        Boolean passingOnly=true
-        Boolean requireMatchingGenotypes=true
-        String vcfScoreField = "QUAL"
```

Comment:
Could we also port the score field into the new arguments?

Reply:
Good idea, just put the `score_field` as an input.